pom.xml

```diff
@@ -469,6 +469,14 @@
                 </configuration>
             </plugin>
 
+            <plugin>
+                <groupId>com.github.spotbugs</groupId>
+                <artifactId>spotbugs-maven-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
+                </configuration>
+            </plugin>
```

Comment:
Do we really want do disable it completely? Can't we add exceptions for just the two files in question?

Reply:
Sure, we can add exceptions instead, but I'm challenging the value of spotbugs for this example project:

> Since this repo doesn't contain any production code (just examples),
we can just disable spotbugs. We still have compiler warnings as
errors and checkstyle.

Reply:
Note that we never _decided_ to add spotbugs to begin with, we just inherited it transitively through the parent poms.

Reply:
Well, writing good example code seems valuable, too :)

Don't have a strong opinion, but if it's company policy to use spotbugs, we might just align to it? \cc @guozhangwang 

Just sharing my opinion. I am also ok to completely disable it if we think that's best.

Reply:
Thanks, @mjsax . There haven't been any further comments, so I'll just go ahead with this. We can always refine it later, but I'd rather not risk forgetting about this PR and releasing broken examples.